libr/search/search.c

```diff
@@ -9,6 +9,12 @@
 
 R_LIB_VERSION (r_search);
 
+typedef struct {
+	ut64 end;
+	int len;
+	ut8 data[];
```

Comment:
`ut8 *data` (avoiding flexible array member) leads to one extra `malloc`, more error check with no benefit.